src/lake/test/env/test.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+
+LAKE=${LAKE:-../../build/bin/lake}
+
+# Test the functionality of `lake env`
+# Also test https://github.com/leanprover/lake/issues/179
+
+# Test `env` with no command
+$LAKE env
+
+# Test installation variables are set
```

Comment:
```suggestion
# Test installation variables are set
# Note: `set -euxo pipefail` above ensures these fail if the variable is not set.
```
Maybe not necessary, but it took me a moment to understand how this was actually testing anything.